server/handlers/tunnel.go

```diff
@@ -44,6 +45,11 @@ var (
 
 // Tunnel forwards the request to the service provider's backend
 func InitTunnel(w http.ResponseWriter, r *http.Request) {
+
+	r.Header.Set("BackendURL", "victus_backend.com")
```

Comment:
is it for testing purpose?

Reply:
Yes, it is testing  value, we can send any backend url instead of it